Test share-url-invalid.https.html should be enabled on iOS-wk2

marcoscaceres opened the pull request:
#### 2c61b57a64a84466864cb999083296bd5cb2e167
<pre>
Test share-url-invalid.https.html should be enabled on iOS-wk2
<a href="https://bugs.webkit.org/show_bug.cgi?id=243769">https://bugs.webkit.org/show_bug.cgi?id=243769</a>

Reviewed by Chris Dumez.

* LayoutTests/platform/ios-wk2/TestExpectations:
* LayoutTests/platform/ios-wk2/imported/w3c/web-platform-tests/web-share/share-url-invalid.https-expected.txt:

Canonical link: <a href="https://commits.webkit.org/253319@main">https://commits.webkit.org/253319@main</a>
</pre>

cdumez reviewed (approved):
LGTM assuming the bots are happy.

marcoscaceres committed:
Test share-url-invalid.https.html should be enabled on iOS-wk2
https://bugs.webkit.org/show_bug.cgi?id=243769

Reviewed by Chris Dumez.

* LayoutTests/platform/ios-wk2/TestExpectations:
* LayoutTests/platform/ios-wk2/imported/w3c/web-platform-tests/web-share/share-url-invalid.https-expected.txt:

Canonical link: https://commits.webkit.org/253319@main
